src/website/app/pages/Theming/theming.md

```diff
@@ -69,16 +69,44 @@ const MyButton = createThemedComponent(Button, {
 
 ### Create your own theme
 
-Use [createTheme](#common-scenarios-api) in order to create a custom theme.  
-Once created, this theme can be applied using a ThemeProvider.
+Use [createTheme](#common-scenarios-api) in order to create a custom theme. Once
+created, this theme can be applied using a ThemeProvider.
+
+#### Use a custom color palette
+
+Each theme variable with a color value derives that value from a color ramp in
+[Mineral UI's palette](/color#guidelines-ramps). Those colors can be derived
+from a different palette color (e.g. "danger" variables use `'bronze'` instead
+of `'red'`, below) or a custom color ramp can be used (e.g. `myAppColor` is used
```

Comment:
For "custom color ramp," it would be useful to hyperlink it or add a sidenote ("see guidelines below") with a link to note or the ramp parameter or the whole section on parameters.